src/components/Header/Header.js

```diff
@@ -1,8 +1,31 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import './Header.css'
 import { Route, Link } from 'react-router-dom';
+import { gsap } from 'gsap';
 
-const Header = ( clearPokemon ) => {
+const Header = ({ foundPokemon }) => {
+
+  useEffect(()=> {
+    gsap.fromTo('.pokeball', 
+    { 
+      opacity: 1,
+      // scale: -3,
+      duration: 2, 
+      ease: "rough({ template: none.out, strength: 1, points: 20, taper: 'none', randomize: true, clamp: false})", 
+      y: 0, 
+      scale: 2.5,
+      rotation: 720
+    },
+    {   
+        scale: 1,
+        opacity: 1,
+        duration: 2.5, 
+        ease: "rough({ template: none.out, strength: 1, points: 20, taper: 'none', randomize: true, clamp: false})", 
+        y: 0,
+       
+        rotation: 0
```

Comment:
This is such a cool effect! I like how you made the ball look like it was in reverse